clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -18,6 +18,7 @@ package slos
 
 import (
 	"fmt"
+	"gopkg.in/yaml.v2"
```

Comment:
Please move this import below, where other, non-single word, imports are

Reply:
Done.